documentacion/como_empezar/README.md

```diff
@@ -73,7 +73,7 @@ pip install transbank-sdk
 
 Te recomendamos leer [las instrucciones de instalación detalladas para el SDK Python](https://github.com/TransbankDevelopers/transbank-sdk-python#instalaci%C3%B3n) para mas opciones de instalación.
 
-(Para webpay en Python puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-python), pero te recomendamos usar [python-tbk, creada por Cornershop](https://github.com/cornershop/python-tbk) que será la base de lo que integremos finalmente en transbank-sdk)
+(Para Webpay en Python puedes seguir usando [libWebpay](https://github.com/TransbankDevelopers/libwebpay-python), pero te recomendamos usar [python-tbk, creada por Cornershop](https://github.com/cornershop/python-tbk) que será la base de lo que integremos finalmente en transbank-sdk)
```

Comment:
I don't think that `libWebpay` is right. Pretty sure it's called `libwebpay` all lowercase